recipes/tab-transformer-pytorch/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "tab-transformer-pytorch" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tab-transformer-pytorch-{{ version }}.tar.gz
+  sha256: 4f8cadb411247f9573240aa81233d533746cb5014a215673566da7ae6cfe15f0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
This one can probably be noarch. If Windows fails due to a missing dependency, it is fine.